drivers/scheduler/k8s/k8s.go

```diff
@@ -627,9 +631,14 @@ func decodeSpec(specContents []byte) (runtime.Object, error) {
 			return nil, err
 		}
 
+		if err := kubevirtv1.AddToScheme(schemeObj); err != nil {
+			return nil, err
+		}
+
 		codecs := serializer.NewCodecFactory(schemeObj)
 		obj, _, err = codecs.UniversalDeserializer().Decode([]byte(specContents), nil, nil)
 		if err != nil {
+			log.Infof("Error occurred after adding kubevirt scheme")
```

Comment:
How about we include err in the message?

Reply:
Actually it is not needed. Let me remove it.